packages/mobile-app/app/onboarding/index.tsx

```diff
@@ -1,27 +1,61 @@
-import { StatusBar } from "expo-status-bar";
-import { StyleSheet, View, Text } from "react-native";
+import { View, StyleSheet } from "react-native";
 import { LinkButton } from "../../components/LinkButton";
+import { VStack, Text } from "@ironfish/tackle-box";
+import { LinearGradient } from "expo-linear-gradient";
+
+import LogoWithText from "../../assets/images/logo-with-text.svg";
+import KeyPlantOrchid from "../../assets/images/key-plant--orchid.svg";
+
+const styles = StyleSheet.create({
+  background: {
+    position: "absolute",
+    top: 0,
+    left: 0,
+    right: 0,
+    bottom: 0,
```

Comment:
suggestion: `StyleSheet.absoluteFill` may come in handy when you're doing this sort of thing